recipes/newrelic/apm/dotNet/windows/net-install.yml

```diff
@@ -49,7 +49,7 @@ install:
           $iisEnabled = Get-WindowsOptionalFeature -Online -FeatureName IIS-WebServer | select State
           if($iisEnabled.State -ne "Enabled"){
             Write-Error "IIS is not installed."
-            exit 1
+            exit 2
```

Comment:
@gabeobrien actually, we may have to update the doc, we want a different error code so we can distinguish between "IIS is not installed" and "ASP.NET is not installed". If we do this change we wouldn't know the difference
cc @jaffinito

Reply:
Agreed.  Updated!